db/hints/manager.hh

```diff
@@ -297,10 +297,12 @@ private:
         _state.set(state::stopping);
```

Comment:
Note that the PR letter will be lost for single-commit pull request when it's merged, only the commit message will remain.

So you will lose parts of the description and `Fixes` part.

You should combine commit message and PR letter and put all of this both into commit message and the letter.